sycl/test/basic_tests/image_accessor_readwrite_half.cpp

```diff
@@ -0,0 +1,162 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUNx: %CPU_RUN_PLACEHOLDER %t.out // OpenCL CPU backend does not support half datatype reads and writes for 1D images.
```

Comment:
Why? Is there is some extension we can check and decide if we need to skip test or not?